cmd/antrea-controller/controller.go

```diff
@@ -89,10 +95,17 @@ func run(o *Options) error {
 
 	go networkPolicyController.Run(stopCh)
 
-	go apiServer.GenericAPIServer.PrepareRun().Run(stopCh)
+	preparedAPIServer := apiServer.GenericAPIServer.PrepareRun()
+	antctl.CommandList.ApplyToMux(preparedAPIServer.GenericAPIServer.Handler.NonGoRestfulMux, nil, controllerMonitor)
+	go preparedAPIServer.Run(stopCh)
+
+	antctlStoppedCh := antctl.StartServer(antctlServerOpt, antctl.CommandList, nil, controllerMonitor, stopCh)
```

Comment:
Why we need the input parameter "antctl.CommandList", we cannt get it from antctl directly?

Reply:
Make sense, I will remove this parameter.